--- sqlite_fdw.spec.old	2024-10-02 18:44:16.785204595 +0700
+++ sqlite_fdw.spec	2024-10-02 18:17:31.920179643 +0700
@@ -15,13 +15,14 @@
 
 Summary:	SQLite Foreign Data Wrapper for PostgreSQL
 Name:		%{sname}_%{pgmajorversion}
-Version:	2.4.0
-Release:	1PGDG%{?dist}
+Version:	%{?release_version}
+Release:	%{?package_release_version}.%{?dist}
 License:	PostgreSQL
 URL:		https://github.com/pgspider/%{sname}
-Source0:	https://github.com/pgspider/%{sname}/archive/v%{version}.tar.gz
+Source0:	sqlite_fdw.tar.bz2
 BuildRequires:	postgresql%{pgmajorversion}-devel pgdg-srpm-macros
-BuildRequires:	postgresql%{pgmajorversion}-server sqlite-devel
+BuildRequires:	postgresql%{pgmajorversion}-server
+BuildRequires:	sqlite >= 3.42.0
 Requires:	postgresql%{pgmajorversion}-server
 %if 0%{?fedora} >= 27
 Requires:	sqlite-libs
@@ -65,11 +66,11 @@
 
 %build
 
-USE_PGXS=1 PATH=%{pginstdir}/bin/:$PATH %{__make} %{?_smp_mflags}
+USE_PGXS=1 PATH=%{pginstdir}/bin/:$PATH %{__make} %{?_smp_mflags} BITCODE_CFLAGS="-fno-strict-aliasing -fwrapv -Xclang -Wno-unused-command-line-argument -Wno-compound-token-split-by-macro -O2"
 
 %install
 %{__rm} -rf %{buildroot}
-USE_PGXS=1 PATH=%{pginstdir}/bin/:$PATH %{__make} %{?_smp_mflags} install DESTDIR=%{buildroot}
+USE_PGXS=1 PATH=%{pginstdir}/bin/:$PATH %{__make} %{?_smp_mflags} install DESTDIR=%{buildroot} BITCODE_CFLAGS="-fno-strict-aliasing -fwrapv -Xclang -Wno-unused-command-line-argument -Wno-compound-token-split-by-macro -O2"
 # Install README and howto file under PostgreSQL installation directory:
 %{__install} -d %{buildroot}%{pginstdir}/doc/extension
 %{__install} -m 644 README.md %{buildroot}%{pginstdir}/doc/extension/README-%{sname}.md
